pkg/hintrunner/zero/zerohint.go

```diff
@@ -122,8 +122,8 @@ func GetHintFromCode(program *zero.ZeroProgram, rawHint zero.Hint, hintPC uint64
 		return createVerifyZeroHinter(resolver)
 	case divModNPackedDivmodV1Code:
 		return createDivModNPackedDivmodV1Hinter(resolver)
-	case isZeroPackCode:
-		return createIsZeroPackHinter(resolver)
+	case isZeroNondetCode:
+		return createIsZeroNondetHinter()
```

Comment:
there is an issue here, you dont call the right create function

Reply:
Right, Fixed it.